### DISCUSSION TITLE
`brew update` in crontab sends output when directed to dev/null

### DISCUSSION THREAD

**Discussion by TravisSpangle** [Homebrew/discussions#84] [Everyday usage] [answered]
I have my crontab configured to mail me when there is output. This helps alert me to failures or reports when I want them.

Recently something changed. I now get mail whenever `brew update` is called. This is my config

` 2 * * * brew update > /dev/null 2>&1`

And the mail I get is 

`Updating Homebrew...`

How do I silent this output?

**Comment by jonchang** [comment, marked as answer]
This isn't a Homebrew issue, but the order of your redirections is incorrect. I'd recommend you read the bash manual section on redirections to understand why this the case: https://www.gnu.org/software/bash/manual/html_node/Redirections.html

**Reply by TravisSpangle** [reply]
Thank you @jonchang !

**Reply by mfriedenhagen** [reply]
BTW: I use something like "command 2>&1 | logger -t command-from-crontab" 
Then you may lookup output in syslog but are not spammed ☺️
